src/core/Akka.Cluster.Tests/MultiNode/ClusterDeathWatchSpec.cs

```diff
@@ -68,55 +77,314 @@ private ClusterDeathWatchSpec(ClusterDeathWatchSpecConfig config)
+                    EnterBarrier("watch-established");
+                    ExpectMsg(path2);
+                    ExpectNoMsg(TimeSpan.FromSeconds(2));
+                    EnterBarrier("second-terminated");
+                    MarkNodeAsUnavailable(GetAddress(_config.Third));
+                    AwaitAssert(() => Assert.True(ClusterView.UnreachableMembers.Select(x => x.Address).Contains(GetAddress(_config.Third))));
+                    Cluster.Down(GetAddress(_config.Third));
+                    //removed
+                    AwaitAssert(() => Assert.False(ClusterView.Members.Select(x => x.Address).Contains(GetAddress(_config.Third))));
+                    AwaitAssert(() => Assert.False(ClusterView.UnreachableMembers.Select(x => x.Address).Contains(GetAddress(_config.Third))));
+                    ExpectMsg(path3);
+                    EnterBarrier("third-terminated");
+                }, _config.First);
 
-            RunOn(() =>
+                RunOn(() =>
+                {
+                    Sys.ActorOf(BlackHoleActor.Props, "subject");
+                    EnterBarrier("subjected-started");
+                    EnterBarrier("watch-established");
+                    RunOn(() =>
+                    {
+                        MarkNodeAsUnavailable(GetAddress(_config.Second));
+                        AwaitAssert(() => Assert.True(ClusterView.UnreachableMembers.Select(x => x.Address).Contains(GetAddress(_config.Second))));
+                        Cluster.Down(GetAddress(_config.Second));
+                        //removed
+                        AwaitAssert(() => Assert.False(ClusterView.Members.Select(x => x.Address).Contains(GetAddress(_config.Second))));
+                        AwaitAssert(() => Assert.False(ClusterView.UnreachableMembers.Select(x => x.Address).Contains(GetAddress(_config.Second))));
+                    }, _config.Third);
+                    EnterBarrier("second-terminated");
+                    EnterBarrier("third-terminated");
+                }, _config.Second, _config.Third, _config.Fourth);
+
+                RunOn(() =>
+                {
+                    EnterBarrier("subjected-started");
+                    EnterBarrier("watch-established");
+                    EnterBarrier("second-terminated");
+                    EnterBarrier("third-terminated");
+                }, _config.Fifth);
+
+                EnterBarrier("after-1");
+            });
+        }
+
+        /* 
+         * NOTE: it's not possible to watch a path that doesn't exist in Akka.NET
+         * REASON: in order to do this, you fist need an ActorRef. Can't get one for
+         * a path that doesn't exist at the time of creation. In Scala Akka they have to use
+         * System.ActorFor for this, which has been deprecated for a long time and has never been
+         * supported in Akka.NET.
+        */
+        //public void AnActorWatchingARemoteActorInTheClusterMustReceiveTerminatedWhenWatchedPathDoesNotExist()
```

Comment:
cc @akkadotnet/developers - need a second opinion on this.

In canonical Akka this portion of the tests requires an `ActorRef` for a non-existent path, creating using `ActorFor` - which has been deprecated within Akka for many versions now and has never been supported in Akka.NET. Therefore I think this test case is more for legacy Akka apps and doesn't apply to Akka.NET.

Can anyone confirm the following for me as a sanity check? (from my code comment)

  /* 
+         * NOTE: it's not possible to watch a path that doesn't exist in Akka.NET
+         * REASON: in order to do this, you fist need an ActorRef. Can't get one for
+         * a path that doesn't exist at the time of creation. In Scala Akka they have to use
+         * System.ActorFor for this, which has been deprecated for a long time and has never been
+         * supported in Akka.NET.
+        */